test-integration/docker-compose.yml

```diff
@@ -0,0 +1,117 @@
+x-common-env-variables: &common-env-variables
+  NETWORK_AUTH_TOKEN: "i_am_very_secret_token"
+  BACALHAU_API_PORT: "1234"
+  MINIO_ROOT_USER: "minioadmin"
+  MINIO_ROOT_PASSWORD: "minioadminpass"
+  AWS_ACCESS_KEY_ID: "minioadmin"
+  AWS_SECRET_ACCESS_KEY: "minioadminpass"
+
+networks:
+  bacalhau-network:
+    driver: bridge
+
+volumes:
+  minio-volume:
+    driver: local
+  registry-volume:
+    driver: local
+
+services:
+  bacalhau-minio-node:
+    image: quay.io/minio/minio
+    container_name: bacalhau-minio-node-container
+    command: server /data --console-address ":9001"
+    volumes:
+      - minio-volume:/data
+    restart: always
+    networks:
+      - bacalhau-network
+    environment: *common-env-variables
+    healthcheck:
+      test: [ "CMD", "curl", "-f", "http://localhost:9000/minio/health/live" ]
+      interval: 1s
+      timeout: 5s
+      retries: 30
+      start_period: 2s
+
+  bacalhau-container-img-registry-node:
+    image: bacalhau-container-img-registry-node-image
+    container_name: bacalhau-container-img-registry-container
+    volumes:
+      - registry-volume:/var/lib/registry
+    restart: always
+    networks:
+      - bacalhau-network
+    environment:
+      REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY: /var/lib/registry
+      REGISTRY_HTTP_ADDR: "0.0.0.0:5000"
+      REGISTRY_HTTP_TLS_CERTIFICATE: "/certs/bacalhau-container-img-registry-node.crt"
+      REGISTRY_HTTP_TLS_KEY: "/certs/bacalhau-container-img-registry-node.key"
+    healthcheck:
+      test: [ "CMD-SHELL", "nc -zv localhost 5000" ]
+      interval: 1s
+      timeout: 5s
+      retries: 30
+      start_period: 2s
+
+  bacalhau-requester-node:
+    image: bacalhau-requester-node-image
+    container_name: bacalhau-requester-node-container
```

Comment:
nit: as we are moving away from the name/term "requester", replacing it with "orchestrator" perhaps we should adopt that name here. @wdbaruni wdyt?

Reply:
I will take a note of that, and if the direction is to move from the "requester" naming to "orchestrator", I will change it in the following PRs on this task.

One point to think about, if we are changing the naming, it would be good to have one commit tha changes everything in the project repo at the same time, this way we do not end up with 2 namings for a long time.